src/data/moves/move-attrs/steal-held-item-chance-attr.ts

```diff
@@ -13,22 +13,15 @@ import { MoveEffectAttr } from "#app/data/moves/move-attrs/move-effect-attr";
  * - "If Knock Off causes a Pokémon with the Sticky Hold Ability to faint, it can now remove that Pokémon's held item."
  */
 export class StealHeldItemChanceAttr extends MoveEffectAttr {
```

Comment:
Should probably remove the `Chance` word from the Attr then

```suggestion
export class StealHeldItemAttr extends MoveEffectAttr {
```